lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country.govspeak.erb

```diff
@@ -84,6 +91,29 @@
     <% if calculator.ceremony_country == 'tunisia' %>
       You’ll need to exchange your UK-issued CNI for one that’s valid in Tunisia at the nearest British embassy to where you’re getting married.
 
+    <% elsif calculator.ceremony_country == 'bulgaria' %>
+
+      ### Exchange your CNI for a locally-issued one
+
+      You must make an appointment at the British embassy in Sofia to to exchange your UK-issued CNI for a locally-issued one.
+
+      [Make an appointment at the embassy in Sofia.](https://www.consular-appointments.service.gov.uk/fco/#!/british-embassy-sofia/certificate-of-no-impediment/slot_picker)
+
+      ### Legalisation and translation
+      Once you have your CNI, you must contact the Bulgarian Ministry of Foreign Affairs to get it legalised (certified as genuine).
+
+      $A
+      2 Alfred Nobel Street
+      Sofia
+      Bulgaria
+      $A
+
+      $C
+      Telephone: (+359) 2 807 64 23, (+359) 2 807 64 28, (+359) 2 807 64 26
+      $C
```

Comment:
In case it helps, here are the three ways the address/contact numbers are being shown:

* [Without any contact numbers](https://github.com/alphagov/smart-answers/blob/all-opposite-sex-outcomes-bulgaria-contd/test/artefacts/marriage-abroad/bulgaria/ceremony_country/partner_british/opposite_sex.txt#L52)
* [With contact numbers on separate lines](https://github.com/alphagov/smart-answers/blob/all-opposite-sex-outcomes-bulgaria-contd/test/artefacts/marriage-abroad/bulgaria/ceremony_country/partner_british/opposite_sex.txt#L115)
* [With contact numbers on the same line](https://github.com/alphagov/smart-answers/blob/all-opposite-sex-outcomes-bulgaria-contd/test/artefacts/marriage-abroad/bulgaria/uk/partner_british/opposite_sex.txt#L28)